docs/Installation/Installation.rst

```diff
@@ -158,4 +160,16 @@ msvc for python.
 
 .. [#anacondaprompt] On OSX or linux this is the command prompt. On Windows, this is accessed from the "Miniconda" or "PYME" folder in the start menu.
 
+Removing a PYME install
+=======================
+
+To upgrade PYME instead of starting fresh, enter ``conda update python-microscopy`` in the Anaconda prompt. Otherwise, read on.
+
+To remove an executable installer on Windows, go to **Start Menu > Settings > Apps**, find `python-microscopy` under
+**Apps & Features**, select it and press *Uninstall*. 
 
+For executable installers on Mac, and conda installations on Windows, Mac and Linux, removing the conda envrionment 
```

Comment:
- executable installers on mac don't install into an evironment
- I think our instructions recommend using `PYME` as the env name

Reply:
* They install into an environment on mine, just defined by path name
* Current instructions use `pyme` http://python-microscopy.org/doc/Installation/Installation.html

Reply:
* I get rid of my packaged install via `conda remove -p /Users/zachcm/python-microscopy -all`

Reply:
Do you recommend switching to `rm -rf /Users/zachcm/python-microscopy`?

Reply:
`conda remove -p /Users/zachcm/python-microscopy -all` will work if you are running conda from another miniconda/anaconda installation, but not if it's the only conda you have installed.

Reply:
Yeah I guess that's the only way I've tried it. So switch to `rm -rf`?